test/test_mps.py

```diff
@@ -7383,7 +7383,7 @@ def helper(value, dim, index, idx_dtype=torch.int32):
     def test_embedding_dense_backward(self):
         def helper(n, d, m, idx):
             embeddingMPS = nn.Embedding(n, d, max_norm=True, device='mps')
-            emedding_weight = embeddingMPS.weight.detach().cpu()
+            embedding_weight = embeddingMPS.weight.detach().cpu()
```

Comment:
It would be nice if this could be landed separately..

Reply:
Reverted